readme.md

```diff
@@ -22,8 +22,8 @@ Hold `Alt` key (`Option` on MacOS) to measure distances in Figma.
 
 ❗️ Replace `<your_account>` with your Github username and copy the links to `Pull Request` description:
 
-- [DEMO LINK](https://<your_account>.github.io/layout_stars/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_stars/report/html_report/)
+- [DEMO LINK](https://YriKm.github.io/layout_stars/)
+- [TEST REPORT LINK](https://YriKm.github.io/layout_stars/report/html_report/)
```

Comment:
Add these links to the PR description 
<img width="934" alt="image" src="https://github.com/user-attachments/assets/7658def6-15d9-4d64-bccf-b87f20870946" />